python/google/protobuf/internal/json_format_test.py

```diff
@@ -1020,6 +1020,80 @@ def testParseEnumValue(self):
         message,
     )
 
+  def testParseUnknownEnumStringValue_Scalar_Proto2(self):
+    message = json_format_pb2.TestNumbers()
+    text = '{"a": "UNKNOWN_STRING_VALUE"}'
+    json_format.Parse(text, message, ignore_unknown_fields=True)
+
+    self.assertFalse(message.HasField('a'))
+
+  def testParseErrorForUnknownEnumValue_ScalarWithoutIgnore_Proto2(self):
+    message = json_format_pb2.TestNumbers()
+    self.assertRaisesRegexp(
```

Comment:
Failure from CI:
https://github.com/protocolbuffers/protobuf/actions/runs/7993753108/job/21830301947?pr=15887#step:4:1822

Reply:
Fixed by using `assertRaisesRegex` (without `p` at the end) which is consistent with the rest of the file.